plugin/gitgutter.vim

```diff
@@ -22,6 +22,7 @@ function! s:set(var, default) abort
   endif
 endfunction
 
+call s:set('g:gitgutter_preview_win_floating',        0)
```

Comment:
It should probably default to use floating windows if supported.

Reply:
I'm happy with that.

Reply:
Sorry I'm confused, should this be true or false?